Add a section to define the URL for the documentation page. (#52)
pyansys/dev-guide: doc/source/guidelines/doc_practices.rst

PyMAPDL provides a self-demonstrating example of how to `add an example reference key <https://mapdldocs.pyansys.com/examples/03-tips-n-tricks/00-example-template.html#ref-how-to-add-an-example-reference-key>`_. 

URL of the documentation page
-----------------------------
To host the documentation related to a PyAnsys project, an easy way to go is to use GitHub Page.
This GitHub feature do not require any database to be setup nor to configure any server.
GitHub will serve the documentation related to the project from a URL tied to PyAnsys organization.

The standard name for this URL can be such as:
   ``<product_name>.docs.pyansys.com``

For instance: `grantami <https://grantami.docs.pyansys.com>`_

> Regarding the documentation dedicated to a specific feature of the product or an example, the URL can be such as:

Reviewer: ```suggestion
Regarding the documentation dedicated to a specific feature of the product or an example, the URL is formatted like this:
```

Author: Great suggestion.

PyMAPDL provides a self-demonstrating example of how to `add an example reference key <https://mapdldocs.pyansys.com/examples/03-tips-n-tricks/00-example-template.html#ref-how-to-add-an-example-reference-key>`_. 

URL of the documentation page
-----------------------------
To host the documentation related to a PyAnsys project, an easy way to go is to use GitHub Page.
This GitHub feature do not require any database to be setup nor to configure any server.
GitHub will serve the documentation related to the project from a URL tied to PyAnsys organization.

The standard name for this URL can be such as:
   ``<product_name>.docs.pyansys.com``

For instance:
`grantami.docs.pyansys.com <https://grantami.docs.pyansys.com>`_

Regarding the documentation dedicated to a specific feature of the product or an example, the URL can be such as:
   ``<extra>.<product_name>.docs.pyansys.com``

For instance:

> `cartpole.mapdl.docs.pyansys.com <https://cartpole.mapdl.docs.pyansys.com>`_

Author: This page is not running yet. I might comment this example for the time being.
